lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -500,7 +515,7 @@ strip_horizontal_space(T, Counter) ->
 
 strip_dot_space(T, Counter, Column) ->
   case strip_horizontal_space(T) of
-    {"#" ++ Rest, _}    -> strip_dot_space(tokenize_comment(Rest), Counter, 1);
+    {"#" ++ Rest, _}    -> strip_dot_space(element(1, tokenize_comment(Rest, [$#], 1)), Counter, 1);
```

Comment:
I intentionally discarded the comment to play it safe as I'm not sure what's the use case to handle comments after a dot. Can you provide an example?